backend/danswer/server/query_and_chat/chat_backend.py

```diff
@@ -90,12 +95,31 @@ def get_user_chat_sessions(
                 time_created=chat.time_created.isoformat(),
                 shared_status=chat.shared_status,
                 folder_id=chat.folder_id,
+                current_alternate_model=chat.current_alternate_model,
             )
             for chat in chat_sessions
         ]
     )
 
 
+@router.put("/update-thread-model")
+def update_thread(
```

Comment:
Also, can we keep the naming consistent across the board here? I think it makes things easier to understand (especially for people less familiar with the code base) if it's called a `ChatSession` across the board rather than `thread` in some places and `ChatSession` in others

Reply:
nit: rename `update_thread` as well